copter/source/docs/traditional-helicopter-manual-tuning.rst

```diff
@@ -0,0 +1,154 @@
+.. _traditional-helicopter-manual-tuning:
+
+===============================
+Manual Tuning Instructions
+===============================
+
+Pitch and Roll Axes
+===================
+Setting _FF and ACCEL_x_MAX Parameters for Desired Pitch and Roll Response
+--------------------------------------------------------------------------
+
+In both pitch and roll axes, the FF gain is set so that the actual aircraft
+rate matches the desired rate. To do this, the RATE message in the log is
+required to compare the RATE.PDes and RATE.P log data for pitch and the RATE.RDes and RATE.R
+log data for roll.
+
+With the FF gains set to 0.15, takeoff and establish a hover
+in Stabilize flight mode, then make some sharp stick inputs in both pitch and
+roll. Land and pull the log from the microSD card and look at the signals in
+a log reviewing software. If the actual rate is more than the desired rate
+then you'll want to decrease FF. If it is less, increase FF.
+
+**(to be added...FF calculation directly from log data)**
+
+If the desired and actual rates are offset by some amount, it means that your swash was not 
+properly leveled in the setup. In this case, just make sure the change in rate is similar in 
+amplitude between desired and actual.  If you get the rates to match and they feel like they 
+are too fast, then reduce the ``ATC_ACCEL_x_MAX`` parameter and repeat the process above to 
+match the desired and actual rates. 
+
+If while tuning the FF gain the aircraft starts to oscillate, reduce the 
+ATC_ANG_xxx_P gain for that axis until the oscillations stop.  However for most 
+helicopters the suggested values above should not cause this problem.
+
+Tuning the D and P gain
+-----------------------
+
+Once you have the heli responding nicely with the rate FF gain, now tune the
+PID gains. The rate PID controller provides stability to reject disturbances and
+keep the actual aircraft following the software demanded rates.
+ 
+Start with the D gain.  Use the :ref:`Transmitter based tuning<common-transmitter-tuning>` feature of ArduCopter.  Make the following parameter changes:
+
++--------------------------------------------+---------+
+| :ref:`TUNE<TUNE>`                          | 21      |
++--------------------------------------------+---------+
+| ``TUNE_LOW`` (prior to 4.0)                | 0       |
```

Comment:
we can probably drop the 4.0 stuff now that we are 4.2, and in all the tables below